src/webxdc.rs

```diff
@@ -625,6 +634,14 @@ impl Context {
         instance_msg_id: MsgId,
         last_known_serial: StatusUpdateSerial,
     ) -> Result<String> {
+        let param = instance_msg_id.get_param(self).await?;
+        if param.get_int(Param::WebxdcIntegration).is_some() {
+            let instance = Message::load_from_db(self, instance_msg_id).await?;
```

Comment:
also thought about that, however, `load_from_db()` load much more data, includes a JOIN, and is not needed for all other webxdc. so, it would be a waste in most cases, adding a second query where there was only one before.

therefore, i added the little, fast lookup before loading the whole thing for the few cases of the integrations.